spec/convertkit/client/subscribers_spec.rb

```diff
@@ -35,12 +36,26 @@ class Client
           tag_id = ENV['TAG_ID']
           email = "crt-subscribers+#{Time.now.to_i}@example.com"
           @client.add_subscriber_to_tag(tag_id, email)
-          
+
           r = @client.unsubscribe(email)
           expect(r.success?).to be_truthy
           expect(r.body).to_not eql({"error"=>"Not Found", "message"=>"The entity you were trying to find doesn't exist"})
         end
       end
+
+      describe "#remove_tag_from_subscriber" do
+        it "removes tag from a subscriber" do
+          tag_id = ENV['TAG_ID']
+          email = "#{SecureRandom.hex}@example.com"
+          subscriber = @client.add_subscriber_to_tag(tag_id, email).body["subscription"]["subscriber"]
+
+          expect {
+            @client.remove_tag_from_subscriber(subscriber["id"], tag_id)
+          }.to change {
+            @client.connection.get("tags/#{tag_id}/subscriptions").body["total_subscriptions"]
```

Comment:
There is a bug in ConvertKit API which is causing this test to fail. Once the number of subscribers is 50 or more then `total_subscriptions` doesn't change and API always says that it is 50. I'll contact ConvertKit regarding this bug.

Reply:
@mrhead did you hear back from convertkit about this? If not, the test could validate that the max number of subscriber returned is 50.

Reply:
Yes I did:

> Hey Patrik,
> thats because that endpoint is paginated with 50 records per page. I've checked API docs and it is not stated there. I've created an issue to update docs so it will be processed soon. Thanks for letting us know!

I told them that using `total_subscriptions` to show number of subscriptions in the current page (vs. number of all subscriptions) doesn't make much sense to me, but I don't expect them to change it.

Testing that max number of returned subscriptions is 50 won't work when you start with a new ConvertKit account. Also, once you reach 50 subscriptions for a tag then it will always return 50 even if you don't do anything.

I'll think about it and maybe I'll come with a better way how to test it.

Reply:
@rkcudjoe I've heard from ConvertKit again and they fixed this and `total_subscriptions` shows total count of all subscriptions in the entire set of subscribers now.

Can you please restart the tests on Semaphore CI? Everything should be green again.